### DISCUSSION TITLE
Feature request: redirect based od 3rd domain level

### DISCUSSION THREAD

**Discussion by lubomier** [YOURLS/YOURLS#3394] [Ideas]
Hello,
i would like to have an option for custom urls (not random hashes) to be redirectable by domain name. 

1. Current behaviour:
yourls.org/mycustomurl -> example.com

2. Desired behaviour
yourls.org/mycustomurl -> example.com
&&
mycustomurl.yourls.org/-> example.com

**Comment by ozh** [comment]
This requires server specific configuration and this is hosting / server dependent.

A possible solution would be to add rewrite rules to map subdomain to subdirectories. See for example https://stackoverflow.com/questions/10642426/htaccess-rewrite-subdomain-to-directory.
